.github/actions/setup/action.yml

```
@@ -1,43 +1,43 @@
-        DISCORD_DEFAULT_GUILD_ID=your_discord_default_guild_id_here
-        OPENAI_API_KEY=sk-your_openai_api_key_here
-        UPSTASH_REDIS_REST_URL=https://your_upstash_redis_rest_url.upstash.io
-        UPSTASH_REDIS_REST_TOKEN=your_upstash_redis_rest_token
-        LOG_LEVEL=INFO
-        LOG_DIRECTORY=logs
-        MEM0_API_KEY=m0-api_key_here
-        DEEPGRAM_API_KEY=your_deepgram_api_key_here
-        ELEVENLABS_API_KEY=your_elevenlabs_api_key_here
-        EXA_API_KEY=your_exa_api_key_here
-        EOF
+name: Check setup
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)